packages/fast-layouts-react/src/utilities/breakpoint-tracker.ts

```diff
@@ -117,7 +138,10 @@ class BreakpointTracker {
         }
 
         this.openRequestAnimationFrame = true;
-        window.requestAnimationFrame(this.update);
+
+        if (canUseDOM()) {
```

Comment:
I don't think this is necessary, we only requestFrame when canUseDOM() is true as it is only assigned in the constructor after that check has already passed.

Reply:
great point, will remove check